### ISSUE TITLE
NetBox API swagger docs fetch url error

### ISSUE THREAD

**Issue by cybarox** [netbox-community/netbox#7401]
### NetBox version

v3.0.4

### Python version

3.8

### Steps to Reproduce

1. open https://demo.netbox.dev/api/docs/
2. View Fetch Error

### Expected Behavior

API docs opens

### Observed Behavior

API docs show Fetch URL error

**Comment by tirkarthi** [comment]
https://github.com/Julian/jsonschema/issues/845#issuecomment-931282776

4.0.0 has now been yanked and 4.0.1 was released with a fix for `python_requires`

**Comment by jeremystretch** [comment]
Well that was fast! Thanks for the heads up, @tirkarthi! Appreciate the quick fix!

**Comment by cybarox** [comment]
with jsonschema=4.0.1 still the same issue, temp fix works for me

**Comment by jeremystretch** [comment]
Agreed, still seems to be an issue.

**Comment by tirkarthi** [comment]
Is there a traceback showing the error in jsonschema?

**Comment by jeremystretch** [comment]
@tirkarthi here's what I've narrowed it down to:

```
$ python3.7 -m venv testing

$ source testing/bin/activate
(testing) $ pip install swagger_spec_validator
Collecting swagger_spec_validator
  Using cached swagger_spec_validator-2.7.3-py2.py3-none-any.whl (27 kB)
Collecting six
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pyyaml
  Using cached PyYAML-5.4.1-cp37-cp37m-manylinux1_x86_64.whl (636 kB)
Collecting jsonschema
  Using cached jsonschema-4.0.1-py3-none-any.whl (69 kB)
Collecting attrs>=17.4.0
  Using cached attrs-21.2.0-py2.py3-none-any.whl (53 kB)
Collecting pyrsistent!=0.17.0,!=0.17.1,!=0.17.2,>=0.14.0
  Using cached pyrsistent-0.18.0-cp37-cp37m-manylinux1_x86_64.whl (119 kB)
Collecting importlib-metadata; python_version < "3.8"
  Using cached importlib_metadata-4.8.1-py3-none-any.whl (17 kB)
Collecting typing-extensions>=3.6.4; python_version < "3.8"
  Using cached typing_extensions-3.10.0.2-py3-none-any.whl (26 kB)
Collecting zipp>=0.5
  Using cached zipp-3.6.0-py3-none-any.whl (5.3 kB)
Installing collected packages: six, pyyaml, attrs, pyrsistent, typing-extensions, zipp, importlib-metadata, jsonschema, swagger-spec-validator
Successfully installed attrs-21.2.0 importlib-metadata-4.8.1 jsonschema-4.0.1 pyrsistent-0.18.0 pyyaml-5.4.1 six-1.16.0 swagger-spec-validator-2.7.3 typing-extensions-3.10.0.2 zipp-3.6.0
WARNING: You are using pip version 20.1.1; however, version 21.2.4 is available.
You should consider upgrading via the '/home/jstretch/testing/bin/python3.7 -m pip install --upgrade pip' command.

$ python
Python 3.7.10 (default, Feb 20 2021, 21:17:23) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from swagger_spec_validator import ref_validators
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jstretch/testing/lib/python3.7/site-packages/swagger_spec_validator/__init__.py", line 8, in <module>
    from swagger_spec_validator.util import validate_spec_url
  File "/home/jstretch/testing/lib/python3.7/site-packages/swagger_spec_validator/util.py", line 9, in <module>
    from swagger_spec_validator import validator12
  File "/home/jstretch/testing/lib/python3.7/site-packages/swagger_spec_validator/validator12.py", line 29, in <module>
    from swagger_spec_validator.ref_validators import default_handlers
  File "/home/jstretch/testing/lib/python3.7/site-packages/swagger_spec_validator/ref_validators.py", line 14, in <module>
    from jsonschema.compat import iteritems
ModuleNotFoundError: No module named 'jsonschema.compat'
>>> 
```

**Comment by tirkarthi** [comment]
https://github.com/Yelp/swagger_spec_validator/pull/146 seems to fix this.

**Comment by jeremystretch** [comment]
I should have this fixed for us by pinning `jsonschema==3.2.0` for now. We should revisit this once `swagger_spec_validator` has been updated.
